app/views/manage_assessments/outcome_coverages/_outcome_coverage.html.erb

```diff
@@ -4,3 +4,10 @@
 <p class="class-card-outcome class-card-outcome-nickname">
   <%= outcome_coverage.outcome_nickname %>
 </p>
+<% if outcome_coverage.assignment.present? %>
+  <p> In <%= outcome_coverage.assignment.name %> on <%= outcome_coverage.assignment.problem %>
```

Comment:
What do you think about using I18n for this?

`t(".assignment", name: outcome_coverage.name, problem: outcome_coverage.problem)`